examples/asr/README.md

```diff
@@ -6,7 +6,7 @@ source env.sh
 bash examples/asr/$task_name/run.sh
```

Comment:
I'd suggest change "Currently supported ASR tasks are HKUST and AISHELL. An example script for Librispeech will be released shortly. To perform the full procedure of ASR experiments, simply run:" to the following:

"Please refer to [the results table](#results-for-asr-tasks) for supported tasks/examples. To run an ASR example, execute the following commands from your Athena root directory:"

This way, we don't have to modify this paragraph to reflect the updated examples.

Reply:
Thanks for the useful comments! I've modified README accordingly.